apidoc/Titanium/UI/WebView.yml

```diff
@@ -432,6 +432,48 @@ methods:
     returns:
         type: BackForwardList
 
+  - name: createPdf
```

Comment:
consider naming `createPDF` instead. Feels odd to lowercase a well-known file extension/acronym that everyone always types out as `PDF`

Reply:
Sounds good. Updated!